src/applications/disability-benefits/all-claims/components/AutoComplete.jsx

```diff
@@ -0,0 +1,191 @@
+      return;
+    }
+
+    debounceSearch(inputValue);
+  };
+
+  const scrollIntoView = index => {
+    const activeResult = resultsRef.current[index];
+    if (activeResult) {
+      activeResult.scrollIntoView({
+        block: 'nearest',
+        behavior: 'smooth',
+      });
+    }
+  };
+
+  const navigateList = (e, direction) => {
+    e.preventDefault();
+    const newIndex =
+      direction === 'down'
+        ? Math.min(activeIndex + 1, results.length - 1)
+        : Math.max(activeIndex - 1, 0);
+    setActiveIndex(newIndex);
+    setActiveResultAnnouncement(
+      `${results[newIndex]}, (${newIndex + 1} of ${results.length})`,
+    );
+    scrollIntoView(newIndex);
+  };
+
+  const selectResult = result => {
+    setValue(result);
+    onChange(result);
+    closeList();
+  };
+
+  const handleKeyDown = e => {
+    if (e.key === 'ArrowDown') {
+      navigateList(e, 'down');
+    } else if (e.key === 'ArrowUp') {
+      navigateList(e, 'up');
+    } else if (e.key === 'Enter' && results.length) {
+      selectResult(results[activeIndex]);
+    } else if (e.key === 'Escape') {
+      closeList();
+    }
+  };
+
+  return (
+    <div className="cc-autocomplete">
+      <VaTextInput
+        label={label}
+        required
+        value={value}
+        onInput={handleInputChange}
+        onKeyDown={handleKeyDown}
+        onBlur={() => setTimeout(closeList, 100)}
+        onFocus={() => {
+          if (value) {
+            debounceSearch(value);
+          }
+        }}
+        message-aria-describedby={!value ? INSTRUCTIONS : null}
+        data-testid="autocomplete-input"
+      />
+      {isOpen &&
+        results.length > 0 && (
+          <>
+            <ul
+              className="cc-autocomplete__list"
+              aria-hidden="true"
+              data-testid="autocomplete-list"
+            >
+              {results.map((result, index) => (
+                <li
+                  key={result}
+                  ref={el => {
+                    resultsRef.current[index] = el;
+                  }}
+                  onClick={() => selectResult(result)}
+                  onKeyDown={handleKeyDown}
+                  className={`cc-autocomplete__option ${
+                    activeIndex === index
+                      ? 'cc-autocomplete__option--active'
+                      : ''
+                  }`}
+                  onMouseEnter={() => setActiveIndex(index)}
+                  role="option"
+                  aria-selected={activeIndex === index}
+                  data-testid={`autocomplete-option-${index}`}
+                >
+                  {index === 0 ? (
+                    <>
+                      Enter your condition as "<strong>{result}</strong>"
+                    </>
+                  ) : (
+                    result
+                  )}
+                </li>
+              ))}
+            </ul>
+            <p role="alert" className="vads-u-visibility--screen-reader">
```

Comment:
We'll want to add back in `aria-live="polite" aria-atomic="true"` here and below. Otherwise every time you land on a dropdown result you get an "alert" sound.